### ISSUE TITLE
Follow up tasks for after Dec 20

### ISSUE THREAD

**Issue by autumnfjeld** [Automattic/wp-calypso#97430]
Project Thread: 
Tracking items we need to circle back on.  Anyone can add items here. 

Please review this list with @oskosk  on Dec 23 and decide what work should be done before the CfT and what work can be punted to the next iteration. 

#### ❗ Must do before shipping to customers
- [x]  [Task: Launchpad task "Select a design" should lead to the correct version of the Design Picker #97717](https://github.com/Automattic/wp-calypso/issues/97717)
#### Maybe do
- [x] DIFM Intro screen 
  - [x] https://github.com/Automattic/wp-calypso/issues/97466
- [ ] Import intro screen 
  - [ ] https://github.com/Automattic/wp-calypso/issues/97454
- [x] Better implementation of Launchpad "Select a design" task when user has been defaulted to 2024 theme when selecting free plan p1734490390911029/1734488253.500379-slack-C085HCWCEDN [{TBD} Handling paid theme choice followed by free plan choice #97624](https://github.com/Automattic/wp-calypso/issues/97624)
- [x] https://github.com/Automattic/wp-calypso/issues/97570#top

## Likely punted to January (next iterations)

- [ ] Follow up on any mobile issues
  - [ ] Floating navigation should be at the bottom p1734382444752689/1734331051.113799-slack-C085HCWCEDN
    - [ ] Work already done https://github.com/Automattic/wp-calypso/pull/97503
    - [ ] Will we need to add branching so flows that which switch from `/start` to `/setup` have consistent navigation? p1734354640236159/1734331051.113799-slack-C085HCWCEDN
- [ ] Bring back "Skip to Dashboard"
- [ ] Big Sky to the goals first flow https://github.com/Automattic/wp-calypso/issues/96758
- [ ] Move the BigSky/Design picker choice step before plans selection https://github.com/Automattic/wp-calypso/issues/96757

**Comment by autumnfjeld** [comment]
I am closing this issue since we are moving to a new phase of post-ship iteration.

Open issues moved to https://github.com/Automattic/wp-calypso/issues/97885


Noting that Big Sky won't be incorporated into Goals-first for a few weeks.  Likely in Feb 2025.
